docs/release_notes.md

```diff
@@ -4,14 +4,44 @@
 
 For a list of all changes in this release, see the [full changelog](https://github.com/nci/scores/compare/1.2.0...develop). Below are the changes we think users may wish to be aware of.
 
+### Introduced Support for Python 3.13 and Dropped Support for Python 3.9
+
+- Introduced support for Python 3.13 and dropped support for Python 3.9. See [PR #710](https://github.com/nci/scores/pull/710).
+
 ### Features
+
+- Added four new metrics:
+	- Quantile Interval Score: `scores.continuous.quantile_interval_score`. See [PR #704](https://github.com/nci/scores/pull/704).
+	- Interval Score: `scores.continuous.interval_score`. See [PR #704](https://github.com/nci/scores/pull/704).
+	- Kling-Gupta Efficiency (KGE): `scores.continuous.kge`. See [PR #679](https://github.com/nci/scores/pull/679) and [PR #700](https://github.com/nci/scores/pull/700). 
+	- Interval threshold weighted continuous ranked probability score (twCRPS) for ensembles: `scores.probability.interval_tw_crps_for_ensemble`. See [PR #682](https://github.com/nci/scores/pull/682).
+- Added an optional `decomposition` argument to several continuous ranked probability score (CRPS) functions. If supplied, the `decomposition` argument will return the under-forecast and over-forecast components of the score in addition to the overall CRPS value. This applies to the following CRPS functions:
+	- continuous ranked probability score (CRPS) for ensembles: `scores.probability.crps_for_ensemble`
+	- threshold weighted continuous ranked probability score (twCRPS) for ensembles: `scores.probability.tw_crps_for_ensemble`
+	- tail threshold weighted continuous ranked probability score (twCRPS) for ensembles: `scores.probability.tail_tw_crps_for_ensemble`
+	- interval threshold weighted continuous ranked probability score (twCRPS) for ensembles: `scores.probability.interval_tw_crps_for_ensemble`)  
+	See [PR #708](https://github.com/nci/scores/pull/708).
```

Comment:
@nicholasloveday could you please check everything I wrote about the decomposition argument for CRPS functions? Could you please check:
(a) Is what I have written accurate, 
(b) Are there any changes that should be made for clarity, and 
(c) Are you happy with the formatting. 

Please let me know of any changes you would like made.

Reply:
Do you want "under-forecast" or "underforecast", and "over-forecast" or "overforecast". I realised after I wrote this that I think you don't use the hyphen. I am happy to remove the hyphens if you prefer (and/or if that would be more consistent with the documentation).

Reply:
Yes that would be great to mention the spread term!

I don't have a strong opinion on the hyphens. I'll leave that choice up to you.

Reply:
I have updated